community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelStatement.java

```diff
@@ -78,8 +79,10 @@
  */
 public class KernelStatement implements TxStateHolder, Statement, AssertOpen
 {
-    private static final boolean TRACK_STATEMENTS = flag( KernelStatement.class, "trackStatements", true );
+    private static final boolean TRACK_STATEMENTS = flag( KernelStatement.class, "trackStatements", false );
```

Comment:
There is no information about `TRACK_STATEMENTS`, a simple comment would suffice. 
I also think it would be valid to note that `RECORD_STATEMENTS_TRACES` does nothing without `TRACK_STATEMENTS`.